<ui version="4.0" >
 <class>TextWindowSimple</class>
 <widget class="QMainWindow" name="TextWindowSimple" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Text Window</string>
  </property>
  <widget class="QWidget" name="widget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>22</y>
     <width>600</width>
     <height>458</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QTextBrowser" name="textBrowser" >
        <property name="font" >
         <font>
          <family>Courier</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="findLabel" >
          <property name="text" >
           <string>Find:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="findLineEdit" />
        </item>
        <item>
         <widget class="QPushButton" name="findNextButton" >
          <property name="text" >
           <string>Next</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="findPrevButton" >
          <property name="text" >
           <string>Previous</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="filePopup" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="saveAction" />
    <addaction name="printAction" />
    <addaction name="separator" />
    <addaction name="exitAction" />
   </widget>
   <addaction name="filePopup" />
  </widget>
  <action name="exitAction" >
   <property name="text" >
    <string>Close</string>
   </property>
   <property name="iconText" >
    <string>Close</string>
   </property>
   <property name="toolTip" >
    <string>Close the window</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="printAction" >
   <property name="text" >
    <string>Print</string>
   </property>
   <property name="iconText" >
    <string>Print</string>
   </property>
   <property name="toolTip" >
    <string>Print the contents of the window</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="clearAllAction" >
   <property name="text" >
    <string>Clear all</string>
   </property>
   <property name="iconText" >
    <string>Clear all</string>
   </property>
   <property name="toolTip" >
    <string>Clear text window contents</string>
   </property>
  </action>
  <action name="saveAction" >
   <property name="text" >
    <string>Save as...</string>
   </property>
   <property name="iconText" >
    <string>Save as...</string>
   </property>
   <property name="toolTip" >
    <string>Save contents of text window to file</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>QTextBrowser</class>
   <extends>QTextEdit</extends>
   <header>QTextBrowser</header>
  </customwidget>
  <customwidget>
   <class>QMainWindow</class>
   <extends>QWidget</extends>
   <header>qmainwindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
